conf/genomes.config

```diff
@@ -5,54 +5,12 @@
  * Defines reference genomes, without using iGenome paths
  * Can be used by any config that customises the base
  * path using $params.genomes_base / --genomes_base
+ *
+ * CAREFUL: Some o the files might be reuiqred in the CI tests not yet implemented. They should be gradually moved to the test.config. Until then lets keep this file.
```

Comment:
"Some o" and "reuiqred" -> "Some of" and "required", I don't know if this is directly implemented from the template, but might be nice to change?

Reply:
no this is not from the template, but a reminder for us to remove this file also as all CI tests are activated